src/relation_manager.py

```diff
@@ -182,9 +184,9 @@ def get_database_options(self) -> Dict[str, Any]:
             host=host,
             port=port,
             unix_socket=unix_socket,
-            username=credentials.get("username"),
-            password=credentials.get("password"),
-            db_name=raw.get("database"),
```

Comment:
Why it used to work earlier? What has been changed?
Will we have other 3rd-party charm which stopped working the same way?

Reply:
I think the issue was the race for the unit file, I don't think it should affect other clients.